### DISCUSSION TITLE
How to get rid of /en/ in the slug

### DISCUSSION THREAD

**Discussion by LagClear** [requarks/wiki#7458] [Help / Questions]
I have my wiki js setup but I noticed in the URL it has /e/ how can I get rid of it?

**Comment by fnafox** [comment]
That only appears while you are `/e/` editing the page.

**Reply by LagClear** [reply]
Sorry I meant /en/

**Reply by fnafox** [reply]
Make sure that Multilingual Namespacing is turned off in `Admin > Locale` then try rerendering pages in `Admin > Utilities > Content: Rerender All Pages`. However I find that it still remains somewhat inconsistent if multiple locales were at any point enabled previously.

**Reply by LagClear** [reply]
Hey sorry just got back into trying resolve this anyways
![image](https://github.com/user-attachments/assets/fa842271-c615-4a8a-9788-38fdf85b9ae3)
I have it like this and tried rerendering it and still acts the same any other solutions?

**Reply by LagClear** [reply]
This was on by default as well as keep in mind I do have pages active right now.

**Reply by LagClear** [reply]
Okay so interesting I found out it works without /en/ in the URL. I am not sure if that worked before or what but then I noticed the nav bar on the left still navigates it to the /en/ sadly.

**Comment by LagClear** [comment]
Bump if anyone has a solution?
